src/CodeStyle/CSharp/CodeFixes/build/Microsoft.CodeAnalysis.CSharp.CodeStyle.props

```
@@ -0,0 +1,13 @@
+﻿<Project>
+  <!-- 
+    PropertyGroup to disable built-in CodeStyle analyzers from .NET SDK that have the identical IDE rules to those implemented in this package.
+    This props file should only be present in the analyzer NuGet package, it should **not** be inserted into the .NET SDK.
+    In future, if we need to insert a CodeStyle props file into the .NET SDK, we should move out the below property group into a separate props file
+    which is not inserted into the .NET SDK.
+
+    NOTE: Keep this file in sync with 'Microsoft.CodeAnalysis.VisualBasic.CodeStyle.props'
+  -->
+  <PropertyGroup>
+    <EnableStyleAnalyzers>false</EnableStyleAnalyzers>
```

Matching the property name in https://github.com/dotnet/sdk/pull/12968/files#diff-08b79d098f2aa427e290083814a935b1R78
```suggestion
    <EnableCodeStyleAnalyzers>false</EnableCodeStyleAnalyzers>
```